extensions/amp-story/1.0/amp-story-reaction-quiz.js

```diff
@@ -188,14 +180,12 @@ export class AmpStoryReactionQuiz extends AmpStoryReaction {
       ).textContent = `${percentage}%`;
     });
 
-    this.rootEl_.setAttribute(
-      'style',
-      `
-      --option-1-percentage: ${percentages[0]}%;
-      --option-2-percentage: ${percentages[1]}%;
-      --option-3-percentage: ${percentages[2]}%;
-      --option-4-percentage: ${percentages[3]}%;
-    `
-    );
+    // TODO(mszylkowski): split each variable to be applied to the corresponding option.
+    setStyles(this.rootEl_, {
+      '--option-1-percentage': percentages[0] + '%',
+      '--option-2-percentage': percentages[1] + '%',
+      '--option-3-percentage': percentages[2] + '%',
+      '--option-4-percentage': percentages[3] + '%',
+    });
```

Comment:
Needed to not override the current styles (background and accent color variables), so switched to setStyles